guide/contribute/formatting.md

```diff
@@ -0,0 +1,407 @@
+---
+layout: guide
+title: Formatting
+description: Visual examples of formatting options available for content authors.
+nav_order: 11
+parent: Contribute
+permalink: /guide/contribute/formatting/
+image: /assets/images/guide/contribute/formatting-preview.jpg
+---
+
+# Formatting
+
+This page showcases the various formatting and layout options available for content. This allows editors to better understand their toolbox and access reference code. It also allows for designers to see the design system in one place. The design source file is a public Figma community file you can find [here](https://www.figma.com/community/file/862622015964353400/Bitcoin-Designers-site). To improve the design, please start with the Figma file and make a proposal in Slack or Github before implementing.
+
+
+## Basic markdown formatting
+
+We use [Markdown](https://daringfireball.net/projects/markdown/) when writing content. Markdown is a plain-text formatting syntax that helps us better prepare our text for the web. Below you can find an overview of commonly used syntax elements.
+
+### Text formatting
+
+Text can be **bold**, _italic_, or ~~strikethrough~~.
+
+```markdown
+**bold**
+_italic_
+~~strikethrough~~
+```
+There should be whitespace between paragraphs.
+
+### Headers
+
+```markdown
+# h1
+## h2
+### h3
+#### h4
+##### h5
+###### h6
+```
+
+# h1 header
+## h2 header
+### h3 header
+#### H4 header
+##### h5 header
+###### h6 header
+
+### Blockquote
+
+```markdown
+> This is a blockquote text.
+```
+
+> This is a blockquote text.
+
+
+### Links
+
+### External Links
+
+```markdown
+[Link to another page](https://bitcoin.org/bitcoin.pdf).
+```
+
+[Link to another page](https://bitcoin.org/bitcoin.pdf).
+
+#### Internal links
+
+```markdown
+[Linking]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})
+```
+[Linking]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})
+
+### Images
+
+Let's start with a very wide image that extends beyond the content with on desktops. Note how a different images is shown on mobile. This can be used to reformat image content to a portrait format.
```

Comment:
```suggestion
Let's start with a very wide image that extends beyond the content with on desktops. Note how a different image is shown on mobile. This can be used to reformat image content to a portrait format.
```